pkg/collect/mssql.go

```diff
@@ -36,14 +38,25 @@ func (c *CollectMssql) IsExcluded() (bool, error) {
 func (c *CollectMssql) Collect(progressChan chan<- interface{}) (CollectorResult, error) {
 	databaseConnection := DatabaseConnection{}
 
+	connUrl, err := url.Parse(c.Collector.URI)
+	// Parsing the uri should not lead to the collector failing
+	// sql.Open will fail if the uri is invalid.
+	if err == nil {
+		klog.V(2).Infof("Connect to %q MSSQL Database", connUrl.Host)
+	} else {
+		klog.V(2).Info("Connect to MSSQL Database")
+	}
+
 	db, err := sql.Open("mssql", c.Collector.URI)
```

Comment:
Looks like after we open the db connection, then there is no `db.Close()` afterward. Should we add one to make sure it closed after we got row.Scan result.